src/main/java/core/basesyntax/service/strategy/Operation.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.service.strategy;
```

Comment:
not resolved, this is part of the transaction model, or create a separate package `enum` somewhere in `model`/`dto` or just `core.basesyntax`